examples/aggregation.py

```diff
@@ -57,20 +59,23 @@ def __str__(self):
     await event.participants.add(participants[0], participants[1])
     await event.participants.add(participants[0], participants[1])
 
-    tournaments_with_count = await Tournament.all().annotate(events_count=Count('events')
-                                                             ).filter(events_count__gte=1)
+    tournaments_with_count = await Tournament.all().annotate(
+        events_count=Count('events'),
+    ).filter(events_count__gte=1)
     assert len(tournaments_with_count) == 1 and tournaments_with_count[0].events_count == 2
 
     event_with_lowest_team_id = await Event.filter(id=event.id).first().annotate(
         lowest_team_id=Min('participants__id')
     )
     assert event_with_lowest_team_id.lowest_team_id == participants[0].id
 
-    ordered_tournaments = await Tournament.all().annotate(events_count=Count('events')
-                                                          ).order_by('events_count')
+    ordered_tournaments = await Tournament.all().annotate(
+        events_count=Count('events'),
+    ).order_by('events_count')
     assert len(ordered_tournaments) == 2 and ordered_tournaments[1].id == tournament.id
-    event_with_annotation = await Event.all().annotate(tournament_test_id=Sum('tournament__id')
-                                                       ).first()
+    event_with_annotation = await Event.all().annotate(
+        tournament_test_id=Sum('tournament__id'),
+    ).first()
```

Comment:
Yeah, this was some of the formatting that yapf just didn't want to get right for me.